tests/e2e_regression/test_cases.sh

```diff
@@ -9,107 +9,111 @@
+  'blocks                             /v1/consensus/blocks'
+  'bad_account                        /v1/consensus/accounts/oasis1aaaaaaa'
+  'account                            /v1/consensus/accounts/oasis1qp0302fv0gz858azasg663ax2epakk5fcssgza7j'
+  'runtime-only_account               /v1/consensus/accounts/oasis1qphyxz5csvprhnn09r49nuyzl0jdw0wsj5xpvsg2'
+  'delegations                        /v1/consensus/accounts/oasis1qpk366qvtjrfrthjp3xuej5mhvvtnkr8fy02hm2s/delegations'
+  'delegations_to                     /v1/consensus/accounts/oasis1qp0j5v5mkxk3eg4kxfdsk8tj6p22g4685qk76fw6/delegations_to'
+  'debonding_delegations              /v1/consensus/accounts/oasis1qpk366qvtjrfrthjp3xuej5mhvvtnkr8fy02hm2s/debonding_delegations'
+  'debonding_delegations_to           /v1/consensus/accounts/oasis1qp0j5v5mkxk3eg4kxfdsk8tj6p22g4685qk76fw6/debonding_delegations_to'
   # NOTE: entity-related tests are not stable long-term because their output is a combination of
   #       the blockchain at a given height (which is stable) and the _current_ metadata_registry state.
   #       We circumvent this by not fetching from metadata_registry at all, so the same metadata (= none) is always present for the test.
-  'entities                       /v1/consensus/entities'
-  'epochs                         /v1/consensus/epochs'
-  'events                         /v1/consensus/events'
-  'proposals                      /v1/consensus/proposals'
-  'proposal                       /v1/consensus/proposals/2'
-  'votes                          /v1/consensus/proposals/2/votes'
-  'tx_volume                      /v1/consensus/stats/tx_volume'
-  'window_size                    /v1/consensus/stats/tx_volume?window_size_seconds=300&window_step_seconds=300'
-  'nonstandard_window_size        /v1/consensus/stats/tx_volume?window_size_seconds=301&window_step_seconds=300'
-  'active_accounts                /v1/consensus/stats/active_accounts'
-  'active_accounts_window         /v1/consensus/stats/active_accounts?window_step_seconds=300'
-  'active_accounts_emerald        /v1/emerald/stats/active_accounts'
-  'txs                            /v1/consensus/transactions'
-  'validators                     /v1/consensus/validators'
-  'emerald_blocks                 /v1/emerald/blocks'
-  'emerald_txs                    /v1/emerald/transactions'
-  'emerald_events                 /v1/emerald/events'
-  'emerald_events_by_type         /v1/emerald/events?type=sdf'
-  'emerald_tokens                 /v1/emerald/evm_tokens'
-  'emerald_token                  /v1/emerald/evm_tokens/oasis1qpgcp5jzlgk4hcenaj2x82rqk8rrve2keyuc8aaf'
-  'emerald_token_holders          /v1/emerald/evm_tokens/oasis1qpgcp5jzlgk4hcenaj2x82rqk8rrve2keyuc8aaf/holders'
-  'emerald_account_with_rose      /v1/emerald/accounts/oasis1qrt0sv2s2x2lkt9e7kmr2mzxgme8m0pzauwztprl'
-  'emerald_account_with_evm_token /v1/emerald/accounts/oasis1qpwx3ptmvcceqkd4syjmqf9jmdlf90xmuuy0f6y9'
```

Comment:
the main change here was that these test cases were moved to the eden-specific range since they aren't valid in damask. Note that no suitable damask equivalent exists because the damask test range is very early and has no evm_tokens/accounts

Reply:
I'm gonna have to transfer these for #665

Reply:
Ah I'll wait for 665 to go in first :)

Reply:
leme try rebasing that earlier then

Reply:
thanks, merged. continue removing this from here. it's now renamed to common_test_cases.sh. edenTestCases is moved to eden/test_cases.sh, so add them there instead